src/c/user-notifications.js

```diff
@@ -85,31 +87,31 @@ const userNotifications = {
                                                 ' Quero receber atualizações dos projetos'
                                             ),
                                             m('.u-marginbottom-20',
-                                                m('a.alt-link[href=\'#\'][id=\'toggle-notifications\']',
+                                                m('a.alt-link[href=\'javascript:void(0);\']', {
+                                                        onclick: () => ctrl.showNotifications.toggle()
```

Comment:
We can replace this kind of pattern for `onclick: ctrl.showNotifications.toggle`